psi4/src/psi4/occ/postprocessing.cc

```diff
@@ -151,6 +150,9 @@ void OCCWave::mp3_postprocessing() {
     Process::environment.globals["SCSN-MP3 CORRELATION ENERGY"] = Escsnmp3 - Escf;
     Process::environment.globals["SCS-MP3-VDW CORRELATION ENERGY"] = Escsmp3vdw - Escf;
     Process::environment.globals["SOS-PI-MP3 CORRELATION ENERGY"] = Esospimp3 - Escf;
```

Comment:
Are SOS-PI-MP3 and the other additional flavors defined anywhere in the literature? Would'nt the E3_SCALE be different each time?

Reply:
As best as I can tell, they are not defined. Section 2.1 of [Bozkaya's second OMP3 benchmark paper](https://pubs.acs.org/doi/10.1021/ct301078q) defines SCS-OMP3 and SOS-OMP3, and the [first OMP3 benchmark paper](https://aip.scitation.org/doi/10.1063/1.3665134) is where SOS-OMP3 was first defined. See the last paragraph of section III for the opposite spin scaling factor he settled on. Grimme defined [defined SCS-MP3](https://onlinelibrary.wiley.com/doi/full/10.1002/jcc.10320)

As far as I can tell, there's been no examination of what the optimal E3_SCALE is for methods other than SCS-MP3. Of the MP3 series, the only spin scaled variants I can find defined in the literature are SCS-OMP3, SOS-OMP3, and SCS-MP3.

Reply:
That agrees with my knowledge about spin-scaled MP3 methods. So essentially the remaining variants are unpublished. I don't know how others feel about this, but I would suggest to comment them out. To avoid users mistakenly using the resulting numbers.

Reply:
...This is going to create _another_ disagreement with DFOCC, but I think this is the right call, scientifically. I want a core dev's approval (probably @loriab) before I strip out all the unpublished spin-scaling schemes from `occ`... Which there are probably a lot of.

Reply:
Removing unpublished is good by me and in the spirit of the P::e->wfn pass.

Reply:
I'll get on it then, unless some other core dev objects. Yak shaving intensifies.